docs/howtos/locally-published-spm-in-firefox.md

```diff
@@ -0,0 +1,105 @@
+
+### Disabling local development
+The easiest way to disable local development is to simply revert any changes to `firefox-ios/Client.xcodeproj/project.pbxproj`.
+
+However, if there are other changes to the file that you would like to preserve, you can use the same script. To use the same script, you will need to:
+1. Know what version of `rust-components-swift` was used beforehand. You can find this by checking the git diff on `firefox-ios/Client.xcodeproj/project.pbxproj`.
+1. Run:
+    ```bash
+    $ ./rust_components_local.sh --disable <VERSION> ../rust-components-swift
+    ```
+1. Then, make sure to reset packages cache in Xcode. This forces Xcode to remove any previously cached versions of the Rust components.
+    - You can reset package caches by going to `File -> Packages -> Reset Package Caches`
+
+
+> If you happen to change branches in `rust-components-swift`, you will need to disable then re-enable local development. The script is not currently smart enough to switch branches. Alternatively, keep the branch in `rust-components-swift` the same. `rust-components-swift` serves only as a release surface so there is little use to switching branches and pushing changes to it, unless you are changing something related to the release process.
+
+## Using the manual flow
+It's important to note the automated flow runs through those steps in a script, so if possible use the script as it's a tedious manual process. However, if the script is failing or you would like to run the manual process for any other reason follow the following steps.
+
+### Building the xcframework
+To build the [xcframework](https://developer.apple.com/documentation/swift_packages/distributing_binary_frameworks_as_swift_packages) do the following:
+1. In your local checkout of `application-services`, navigate to [`megazords/ios-rust/`](https://github.com/mozilla/application-services/tree/main/megazords/ios-rust)
+1. Run the `build-xcframework.sh` script: 
+```bash
+$ ./build-xcframework.sh
+```
+This will produce a file name `MozillaRustComponents.xcframework.zip` that contains the following, built for all our target iOS platforms.
+- The compiled Rust code for all the crates listed in `Cargo.toml` as a static library
+- The C header files and [Swift module maps](https://clang.llvm.org/docs/Modules.html) for the components
+
+## Include the xcframework in a local checkout of `rust-components-swift`
+After you generated the `MozillaRustComponents.xcframework.zip` in the previous step, do the following to include it in a local checkout of `rust-components-swift`:
+1. Unzip the `MozillaRustComponents.xcframework.zip` into the `rust-components-swift` repository: (Assuming you are in the root of the `rust-components-swift` directory and `application-services` is a neighbor directory)
+    ```sh
+     unzip  ../application-services/megazords/ios-rust/MozillaRustComponents.xcframework.zip -d .
+    ```
```

Comment:
This prompted me about overwriting files - I assume that's expected, so maybe add `-o` to the command-line?

Reply:
Good catch - I'll update it! It's expected if you've gone through the process before (or have run the script which does the whole process)

Reply:
> (or have run the script which does the whole process)

Right, yeah, that needs to be clearer - I paid special attention to the heading levels, so that once we were back to a top-level heading, we were back to "must be done in both cases"